gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageObjectImmutableException.java

```diff
@@ -1,7 +1,10 @@
 package com.google.gcloud.storage.contrib.nio;
 
-/** Exception thrown to indicate we don't support a mutation of a cloud storage object. */
+/**
+ * Exception thrown to indicate we don't support a mutation of a cloud storage object.
```

Comment:
Super nit: we can capitalize "Cloud Storage" (it's lower case in a couple other places, so maybe a search-replace might be helpful)